### ISSUE TITLE
standardizing variables

### ISSUE THREAD

**Issue by GretaKu** [UniLuFS2020-ReplicationSeminar/ReplicationPaperGreta#4]
@DeAngelisA Hi! I hope you had a good weekend! 

I am currently working on processing the panel datasets to one final dataset. The author is using a self-made function to **standardize variables** :
`two_sd <- function(x) {
  return((x - mean(x, na.rm = TRUE))/(2*sd(x, na.rm = TRUE)))
}`

I am a bit confused because I don't understand why it divides by **2***SD. I would have set up the function as follows, according to the z transformation:
`ztrans <- function(x) {
  return(
    (x - mean(x, na.rm = T)) / sd(x, na.rm = T)
  )
}`

By analyzing the two functions I see that two_sd is mostly between -1 and 1, and z transformation between -2 and 2. When do you use which function? What makes the difference by using these different functions?

**Comment by DeAngelisA** [comment]
@GretaKu : standardizing is useful to interpret and compare coefficients. The classic standardization consists of subtracting the mean and dividing by 1 standard deviation. Gelman and Hill (2007), I think in chapter 4 but I might be wrong, suggest dividing by 2 standard deviations because this allows to compare more directly dichotomous and continuous variables. One-standard-deviation change in a dummy variable corresponds approximately to a change of 0.5 (e.g. from 0 to 0.5) which would imply an extrapolation. Dividing by 2 sd instead this would be a change from 0 to 1 (approximately the whole range of variation in the variable) and so you can better compare the predictive differences. 

To see the differences, focus on a dichotomous variable and: 1. run a regression model on the unstandardized variables and interpret the coefficient. 2. repeat with classic standardization. See that the unstandardized coefficients multiplied by the standard deviation of a variable equals the standardized coefficient. 3. do the same with the 2sd standardization. Which one is easier to interpret? Why?

**Comment by GretaKu** [comment]
@DeAngelisA : Thank you for your input. I checked out the book and found as well a study about it, where it is really nicely explained:  [http://www.stat.columbia.edu/~gelman/research/published/standardizing7.pdf](url)

I ran a few examples myself and i noticed how the coefficients for binary variables are easily comparable with variables standardized dividing by 2sd. I will keep this in mind while working on my replication.
